File: back-end/src/modules/wishlist/dtos/product-in-wishlist-response.dto.ts
/**
 * @description Product in wishlist response dto
 * @author Nhut Tan
 * @since 2025-09-23
 * @version 1.0.0
 */

export class ProductInWishlistResponseDto {
	productID: number;
	name: string;
	imageUrl: string;
	price: number;
	discount: number;
	rating: number;
	createdAt: string;
	updatedAt: string;
}

File: ECommerce/app/src/main/java/com/example/e_commerce/DTOs/Response/ProductInWishlistResponseDto.kt
/**
 * @description Product in wishlist response dto
 * @author @nhuttan12
 * @since 2026-01-11
 * @version 1.0.0
 */

package com.example.e_commerce.DTOs.Response

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ProductInWishlistResponseDto(
    val productID: Int,
    val name: String,
    val imageUrl: String,
    val price: Double,
    val discount: Int,
    val rating: Double,
    val createdAt: String,
    val updatedAt: String,
)
